client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -151,7 +65,7 @@ class Unlocked extends React.Component {
 			<Button
 				className="transfer-out__action-button"
 				onClick={ this.handleCancelTransferClick }
-				disabled={ this.state.submitting || ! this.state.sent }
+				disabled={ this.props.isSubmitting || ! this.state.sent }
 			>
 				{ this.props.translate( 'Cancel Transfer' ) }
```

Comment:
For some reason, when the API returns an error (I've just hardcoded a `return new WP_Error` on the backend), this button does not revert to the active state and remains disabled:

<img width="1057" alt="Screenshot 2020-11-18 at 19 01 28" src="https://user-images.githubusercontent.com/3392497/99575465-94201900-29d0-11eb-9850-fa105939bc8a.png">

This looks like a regression, because on production it becomes active on error (in other words, it turns active when there's a response from the server - regardless if it's a success or not):

<img width="1054" alt="Screenshot 2020-11-18 at 19 02 25" src="https://user-images.githubusercontent.com/3392497/99575579-b7e35f00-29d0-11eb-8627-18711084f1e6.png">

There could be legitimately cases where a retry during this process could help, so it'd be best if we could ensure the user does not have to refresh the whole page to do so.

FWIW, the primary button on this page works as expected.

Reply:
Really good catch 💯  Since you don't consider it a blocker, I'll fix it in a follow-up PR as soon as I merge this one.

Reply:
Fix for what you reported is here: #47577 and it could use your 👀 . 

Thanks again @klimeryk 🙌